### ISSUE TITLE
I found a strange part of your code.

### ISSUE THREAD

**Issue by seolhokim** [iliaschalkidis/ELMo-keras#2]
first of all, i'm poor at english.

if you find impolite parts of my sentences, please forgive me generously.

lines 128~134 , in ELMo-keras/elmo/model.py .

re_lstm layer got lstm_inputs layer, but lstm_inputs layer is forward_lstm.

isn't weird? i think you intented to use lstm_inputs, in line 90. i think you have to change name.

anyway, i also implement elmo now, your code is very helpful. thank you so much.

**Comment by iliaschalkidis** [comment]
@seolhokim thanx a lot for reviewing the code and revealing this major bug! I just fixed it. I rewrote and transfer the code from my initial implementation to this repo. Then I probably never tried using the GPU-accelerated method in order to find out this bug... 

I'll give you credits in repo's readme. Let me know if you'll find out anything else that does not feel ok to you ;)

Your english is just fine, I totally understood your point and that's what really matters.
